src/aleph/vm/orchestrator/metrics.py

```diff
@@ -35,10 +43,11 @@ def create_tables(engine: Engine):
 
 
 class ExecutionRecord(Base):
-    __tablename__ = "records"
+    __tablename__ = "executions"
```

Comment:
Why rename the table ? What would the old table become after the upgrade ?
Do we need both a table for the history of executions and for the currently running VMs ?

Reply:
I have changed de table name because it was only used to know the current executions. Wasn't used as a history either before, because was only used on VM stopping. Anyways, the VM record data was stored on files on VM stop also, and we don't expose either that information.
Now that database is just the table to use to store the current running VMs.

Reply:
Wouldn't this require an alembic migration then? Or is this table just too ephemeral for us to care?

Reply:
I didn't see where we use the old execution data, and previously, we were saving it just when the VM stops, so now, after this PR, we maintain the current execution's registry fine. Also, I can maintain the old database to see the already stopped executions, but I don't see the advantage to do it.